.github/workflows/verify-repo-tags.yml

```diff
@@ -0,0 +1,75 @@
+name: "Verify Repo Tags"
+
+on:
+  schedule:
+    - cron:  '0 7 * 1 5'
```

Comment:
Ooh also do we want to change the time to 9 or 10? There's a risk a 7am Slack message will be missed by all the Slack noise.

May also be nice to add an inline comment e.g. `# 9am Mon-Fri`